registry/api/v2/descriptors.go

```diff
@@ -514,7 +514,7 @@ var routeDescriptors = []RouteDescriptor{
 									digestHeader,
 								},
 								Body: BodyDescriptor{
-									ContentType: "application/json; charset=utf-8",
+									ContentType: "<media type of manifest>",
```

Comment:
AFAIK `Name` and `Path` are used to build the url router and URLs.  `APIDescriptor` uses these structs to generate the documentation.